tests/bsim/bluetooth/audio/prj.conf

```
@@ -46,6 +46,7 @@ CONFIG_BT_ISO_TX_BUF_COUNT=4
 CONFIG_BT_ISO_MAX_CHAN=4
 CONFIG_BT_ISO_TX_MTU=310
 CONFIG_BT_ISO_RX_MTU=310
+CONFIG_BT_ISO_RX_BUF_COUNT=4
```

Move the Tx buf count together here.

```suggestion
CONFIG_BT_ISO_TX_BUF_COUNT=4
CONFIG_BT_ISO_RX_BUF_COUNT=4
```

---

Generally I would rather not change more lines than what's needed. 

I would also rather group the Kconfigs by direction, so we get 

```
CONFIG_BT_ISO_TX_MTU=310
CONFIG_BT_ISO_TX_BUF_COUNT=4
CONFIG_BT_ISO_RX_MTU=310
CONFIG_BT_ISO_RX_BUF_COUNT=4
```

What do you think about that?

---

yes, its better.